src/vmq_reg.erl

```diff
@@ -332,7 +328,8 @@ deliver_retained({MP, _} = SubscriberId, Topic, QoS) ->
       end, ok, MP, Topic).
 
 subscriptions_for_subscriber_id(SubscriberId) ->
-    plumtree_metadata:get(?SUBSCRIBER_DB, SubscriberId, [{default, []}]).
```

Comment:
In the `vmq_subscriber_db:read` call we no longer pass the `[{default, []}]` proplist to `plumtree_metadata:get/X`. I suppose that's fine (it doesn't change anything anyway)?